src/seedu/addressbook/Main.java

```diff
@@ -72,13 +78,18 @@ private void start(String[] launchArgs) {
         }
     }
 
-    /** Prints the Goodbye message and exits. */
+    /**
+     * Prints the Goodbye message and exits.
+     */
     private void exit() {
         ui.showGoodbyeMessage();
         System.exit(0);
     }
 
-    /** Reads the user command and executes it, until the user issues the exit command.  */
+    /**
+     * Reads the user command and executes it, until the user issues the exit command.
+     * sad
```

Comment:
Eh what does this mean? Please note that comments should be about the code only. It should still make sense to any future developer reading the comment. If you want to tell something to the reviewer/tutor (but not future developers of the code), use a GitHub comment instead (like this comment I've added).

Reply:
Oh, because i have problem pushing to github, so i spent hours making commits randomly and forget to remove it as i have forgotten.

Reply:
It's okay. Just try to avoid it in future :)